hs_core/management/commands/sync_email_subscriptions.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+
+"""
+Clears out two-day-old zip files from iRODS storage
+Meant to be run once a day via Jenkins
+"""
+
+from datetime import datetime, timedelta
+import json
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+import requests
+
+import logging
+
+from theme.models import UserProfile
+
+
+class Command(BaseCommand):
+    help = "Clears out two-day-old zip files from iRODS storage."
```

Comment:
@aphelionz This help text does not apply to this management command and needs to be updated.